requirements.txt

```diff
@@ -1,16 +1,16 @@
-algoliasearch>=2.0,<3.0
-flake8==3.7.9
-Flask-Limiter==1.1.0
-Flask-Migrate==2.5.2
-Flask-Cors==3.0.8
-psycopg2-binary==2.8.4
-pytest-mock==2.0.0
-pytest-cov==2.8.1
-pyyaml==5.3
-requests==2.22.0
-sqlalchemy==1.3.13
-SQLAlchemy-Utils==0.36.1
-prometheus_client==0.7.1
-uWSGI==2.0.18
-py-healthcheck==1.9.0
-bandit==1.5.1
```

Comment:
We want to keep out dependencies pinned. The problem we're seeing with Werkzueg in production is because that package _isn't_ pinned. Please take this change out

Reply:
want me to revert this entirely (security vulns!) or just bump all the pins?

Reply:
Security vuln resolution sounds like a separate PR

Reply:
Yeah, revert it for now. Feel free to review #271 and check that everything is up to par security wise, and that everything had been included

Reply:
y'all are some party poopers.  reverted.